modules/core/src/index.js

```diff
@@ -75,13 +75,6 @@ export {default as _Attribute} from './core/attribute';
 export {default as _ShaderCache} from './core/shader-cache';
 export {default as _AnimationLoopProxy} from './core/animation-loop-proxy';
 
-// SCENEGRAPH
-export {default as ScenegraphNode} from './scenegraph/scenegraph-node';
-export {default as Group} from './scenegraph/group';
-export {default as Camera} from './scenegraph/camera';
-export {default as Model} from './scenegraph/model';
-export {default as createGLTFObjects} from './scenegraph/gltf/create-gltf-objects';
```

Comment:
There is no longer a Model export from core?

Reply:
> There is no longer a Model export from core?

There will be, this is WIP.

The Model split is the main problem that needs to be solved. Naturally, the whole point is that Model remains in core so it can be used without the scenegraph module. To do that, it must not inherit from ScenegraphNode, so we need a separate ScenegraphNode subclass (that imports and uses Model from core).

And then update examples that depend on Model scenegraph methods, update docs etc.

Should not be particularly difficult, just a bunch of work, will iterate on this for a while.

Reply:
> There is no longer a Model export from core?

Model has now been split!